docs/docs/hpc-on-k8s/hadoop.md

```diff
@@ -0,0 +1,56 @@
+# Hadoop Workload
+Hadoop is an open-source framework that efficiently processes and stores vast amounts of data across distributed computing clusters. It breaks down large datasets into manageable chunks and distributes them across multiple computers, allowing for parallel processing and analysis. This distributed approach makes it possible to handle massive amounts of data that would be impractical to process on a single machine.
+
+
+## Steps to Execute Job
+Below is the example manifest file for Hadoop Mapreduce workload in a Kubernetes cluster.
+```yaml
+# job.yaml
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: hadoop-
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  parallelism: 1
+  completions: 1
+  suspend: true
+  template:
+    spec:
+      containers:
+      - name: dummy-job
+        image: shubhamdang/my_hadoop_image:latest
```

Comment:
Is this still the image repository or has the openflight one been created now?

Reply:
updated and image is present in openflighthpc dockerhub account.